OSOE-530: FeatureCodeGeneration UI test can be flaky with StaleElementReferenceException in Lombiq.HelpfulExtensions

Piedone opened the pull request:
[OSOE-530](https://lombiq.atlassian.net/browse/OSOE-530)
Also fixing similar flakyness in Privacy.

[OSOE-530]: https://lombiq.atlassian.net/browse/OSOE-530?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Piedone committed:
More reliable UI test in Privacy

Piedone committed:
Updating Privacy NuGet to latest

Piedone committed:
More reliable UI test in HelpfulExtensions

Piedone committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-530

Piedone committed:
HelpfulExtensions refactoring

Piedone committed:
Safer assertion in UI test in Privacy

Piedone committed:
Typos

Piedone committed:
Better retries in HelpfulExtensions UI test

Piedone committed:
Fixing that stale retries in UITestingToolbox

Piedone committed:
Updating Helpful Extensions NuGet reference

Piedone committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-530

# Conflicts:
#	test/Lombiq.UITestingToolbox

dministro commented on NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/Lombiq.OSOCE.NuGet.Tests.UI.csproj:
```diff
@@ -24,8 +24,8 @@
-    <PackageReference Include="Lombiq.HelpfulExtensions.Tests.UI" Version="4.0.0" />
+    <PackageReference Include="Lombiq.Privacy.Tests.UI" Version="4.0.1-alpha.6.osoe-530" />
+    <PackageReference Include="Lombiq.HelpfulExtensions.Tests.UI" Version="4.0.1-alpha.4.osoe-530" />
     <PackageReference Include="Lombiq.Tests.UI" Version="5.0.1-alpha.1.osoe-464" />
```
I think we need a prerelease of `Lombiq.Tests.UI` because of the changes related to `RetryIfStaleOrFailAsync`. Without this fix the NuGet tests could fail with `StaleElementReferenceException` which can confusing for other developers.

Piedone replied:
Indeed, but allow me not to do that, since as part of https://lombiq.atlassian.net/browse/OSOE-532 I'll do a release of everything. Releasing an alpha of UITT for this necessitates an update to most of the other projects, which in light of a proper release is a waste of time.

dministro committed:
Updating submodules:
 - Lombiq.HelpfulExtensions: fast-forward
 - Lombiq.Privacy: fast-forward
 - Lombiq.UITestingToolbox: fast-forward
